app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -126,3 +129,95 @@ def test_create_new_experiment_remote_settings(selenium, base_url):
             break
     summary_page = SummaryPage(selenium, base_url).wait_for_page_to_load()
     assert "live" in summary_page.experiment_status.lower()
+
+
+def test_create_new_experiment_remote_settings_reject(selenium, base_url):
+    experiment_name = f"name here remote {random.randint(0, 1000)}"
+
+    selenium.get(base_url)
+    home = HomePage(selenium, base_url).wait_for_page_to_load()
+    home.tabs[-1].click()  # Click drafts
+    current_experiments = len(home.tables[0].experiments)
+    experiment = home.create_new_button()
+    experiment.public_name = experiment_name
+    experiment.hypothesis = "smart stuff here"
+    experiment.application = "DESKTOP"
+
+    # Fill Overview Page
+    overview = experiment.save_and_continue()
+    overview.public_description = "description stuff"
+    overview.select_risk_brand_false()
+    overview.select_risk_revenue_false()
+    overview.select_risk_partner_false()
+
+    # Fill Branches page
+    branches = overview.save_and_continue()
+    branches.remove_branch()
+    branches.reference_branch_name = "name 1"
+    branches.reference_branch_description = "a nice experiment"
+    branches.feature_config = "No Feature Firefox Desktop"
+
+    # Fill Metrics page
+    metrics = branches.save_and_continue()
+
+    # Fill Audience page
+    audience = metrics.save_and_continue()
+    audience.channel = "Nightly"
+    audience.min_version = 80
+    audience.targeting = "US_ONLY"
+    audience.percentage = 50.0
+    audience.expected_clients = 50
+    audience.save_btn()
+    review = audience.save_and_continue()
+
+    # Review and approve
+    selenium.find_element_by_css_selector("#PageRequestReview")
+    review.launch_without_preview.click()
+    review.request_review.click_launch_checkboxes()
+    review.request_review.request_launch_button.click()
+    review.approve()
```

Comment:
Q: Should this be `.action()`?

Reply:
Action is a function for remote settings.